test/test_util.py

```diff
@@ -48,106 +48,96 @@
-            'http://[2a00:1450:4001:c01::67]:80/test': ('http', '[2a00:1450:4001:c01::67]', 80),
-
-            # More IPv6 from http://www.ietf.org/rfc/rfc2732.txt
-            'http://[fedc:ba98:7654:3210:fedc:ba98:7654:3210]:8000/index.html': (
-                'http', '[fedc:ba98:7654:3210:fedc:ba98:7654:3210]', 8000),
-            'http://[1080:0:0:0:8:800:200c:417a]/index.html': (
-                'http', '[1080:0:0:0:8:800:200c:417a]', None),
-            'http://[3ffe:2a00:100:7031::1]': ('http', '[3ffe:2a00:100:7031::1]', None),
-            'http://[1080::8:800:200c:417a]/foo': ('http', '[1080::8:800:200c:417a]', None),
-            'http://[::192.9.5.5]/ipng': ('http', '[::192.9.5.5]', None),
-            'http://[::ffff:129.144.52.38]:42/index.html': ('http', '[::ffff:129.144.52.38]', 42),
-            'http://[2010:836b:4179::836b:4179]': ('http', '[2010:836b:4179::836b:4179]', None),
-        }
-        for url, expected_host in url_host_map.items():
-            returned_host = get_host(url)
-            self.assertEqual(returned_host, expected_host)
-
-    def test_invalid_host(self):
-        # TODO: Add more tests
-        invalid_host = [
-            'http://google.com:foo',
-            'http://::1/',
-            'http://::1:80/',
-            'http://google.com:-80',
-            six.u('http://google.com:\xb2\xb2'),  # \xb2 = ^2
-        ]
-
-        for location in invalid_host:
-            self.assertRaises(LocationParseError, get_host, location)
-
-    def test_host_normalization(self):
-        """
-        Asserts the scheme and hosts with a normalizable scheme are
-        converted to lower-case.
-        """
-        url_host_map = {
-            # Hosts
-            'HTTP://GOOGLE.COM/mail/': ('http', 'google.com', None),
-            'GOogle.COM/mail': ('http', 'google.com', None),
-            'HTTP://GoOgLe.CoM:8000/mail/': ('http', 'google.com', 8000),
-            'HTTP://user:password@EXAMPLE.COM:1234': ('http', 'example.com', 1234),
-            '173.194.35.7': ('http', '173.194.35.7', None),
-            'HTTP://173.194.35.7': ('http', '173.194.35.7', None),
-            'HTTP://[2a00:1450:4001:c01::67]:80/test': ('http', '[2a00:1450:4001:c01::67]', 80),
-            'HTTP://[FEDC:BA98:7654:3210:FEDC:BA98:7654:3210]:8000/index.html': (
-                'http', '[fedc:ba98:7654:3210:fedc:ba98:7654:3210]', 8000),
-            'HTTPS://[1080:0:0:0:8:800:200c:417A]/index.html': (
-                'https', '[1080:0:0:0:8:800:200c:417a]', None),
-            'abOut://eXamPlE.com?info=1': ('about', 'eXamPlE.com', None),
-            'http+UNIX://%2fvar%2frun%2fSOCKET/path': (
-                'http+unix', '%2fvar%2frun%2fSOCKET', None),
-        }
-        for url, expected_host in url_host_map.items():
-            returned_host = get_host(url)
-            self.assertEqual(returned_host, expected_host)
-
-    def test_parse_url_normalization(self):
+class TestUtil(object):
+
+    url_host_map = {
```

Comment:
It doesn't seem like this needs to be a dictionary, and the dictionary nature of it makes it really easy for us to accidentally miss a test.